plugins/plugin-template/src/index.ts

```diff
@@ -0,0 +1,63 @@
+import {
+  DsgContext,
+  AmplicationPlugin,
+  Events,
+  Module,
+  EventNames,
+  CreateServerParams,
+  CreateAdminUIParams,
+} from "@amplication/code-gen-types";
+
+
+class ExamplePlugin implements AmplicationPlugin {
+  /**
+   * this is mandatory function that return object with events name. each event can have before or/and after
+   */
+  register(): Events {
+    return {
+    [EventNames.CreateServer]: {
+      before: this.beforeCreateExample,
```

Comment:
I think we should use real names 
```suggestion
      before: this.beforeCreateServer,
```

Reply:
don't you think it will be confusing ?